src/mscorlib/GenerateCompilerResponseFile.targets

```diff
@@ -16,7 +16,8 @@
       <Clean Include="$(IntermediateOutputPath)\moduleName.rsp" />
     </ItemGroup>
     <PropertyGroup>
-      <CompilerResponseFile>$(IntermediateOutputPath)\moduleName.rsp</CompilerResponseFile>
+      <CompilerResponseFile Condition="'$(CompilerResponseFile)'!=''">$(IntermediateOutputPath)\moduleName.rsp;$(CompilerResponseFile)</CompilerResponseFile>
```

Comment:
Can this be simplified to be something like this:

<CompilerResponseFile>$(IntermediateOutputPath)\moduleName.rsp;$(CompilerResponseFile)</CompilerResponseFile>

Also, does this change needs to account for System.Private.CoreLib that is also being built?